integration_testing/features/super-admin/super-admin-export-content-to-local-drive.feature

```diff
@@ -3,61 +3,68 @@ Feature: Super admin exports content to local drive
-      And I see the *content* subfolder inside 
-      And I see the *databases* and *storage* subfolders inside the *content* folder
+    Given there is a <drive> local drive attached to the device 
+      When I click *Export*
+      Then I see Kolibri searching for local drives
+        And I see the *Select export destination* modal
+      When I select the <drive> local drive
+      Then I see the *Continue* button is active
+      When I click the *Continue* button
+      Then I see the list of channels present on the device
+      When I click the *Select* button for the <channel> channel
+      Then I see *Select content from…* page
+        And I see the list of topics for the <channel> channel
+        And I see the total number and size of <channel> channel resources
+        And I see the value for *Drive space available*      
+        And I see the value for *Content selected* is 0
+      # Select/deselect all the topics
+      When I check the *Select all* checkbox
+      Then I see the checkboxes for all the topics are checked
+        And I see the *Export* button is active
+        And I see the values for *Content selected* increase
+      When I uncheck the *Select all* checkbox
+      Then I see the *Export* button is inactive
+        And I see the values for *Content selected* is 0
+      # Select/deselect one full topic    
+      When I check the <topic> topic checkbox
+      Then I see the *Export* button is active
+        And I see the values for *Content selected* increase
+      When I uncheck the <topic> topic checkbox
+      Then I see the *Export* button is inactive
+        And I see the values for *Content selected* is 0
+      # Select and import just one resource from a subtopic of a topic
+      When I click the <topic> topic
+      Then see the list of subtopics for the <topic> topic
+      When I click the <subtopic> subtopic
+      Then see the list of resources for the <subtopic> subtopic
+      When I check the <resource> resource checkbox
+      Then I see the *Export* button is active 
+        And I see the *1 resource selected* flag for the <resource> resource
+        And I see the values for *Content selected* increase
+      When I click the *Export* button
+      Then I see *Device > Channels* page again
+        And I see the blue progress bar with the percentage increasing 
+      When the export process concludes
+      Then I see the progress bar at 100%
+        And I see the *Finished! Click "Close" button to see changes.* flag
+        And I see the *Close* button
+      When I click *Close* 
+      Then I see the *Content* page is reloaded 
+        And I open the <drive> local drive
+        And I see the *KOLIBRI_DATA* folder on the <drive> local drive
+        And I see the *content* subfolder inside 
+        And I see the *databases* and *storage* subfolders inside the *content* folder
+
+    Scenario: No writable drives found
```

Comment:
How can we achieve this scenario?  I haven't simulated this coz my VM always has the drive C: available. :(

Reply:
@cpauya Not sure how is your VM set up... On mines, when they don't have writable USB devices attached, and no shared folders configured, I can see the notification correctly.

![Selection_157](https://user-images.githubusercontent.com/1457929/61386743-1ff1d480-a8b5-11e9-824d-1a499691ffa6.png)

![Selection_158](https://user-images.githubusercontent.com/1457929/61386763-25e7b580-a8b5-11e9-9874-cde4b1ff44aa.png)

Reply:
Thanks!  Got a similar config, however, in my VirtualBox 6.0.10 VM it's showing `Local Fixed Disk` even though I have no USB device attached, no Shared Folder, and no other Storage Devices aside from the SATA disk for the VM (adding a CD-ROM with an .iso attached shows up too).

![Screenshot 2019-07-17 23 34 38](https://user-images.githubusercontent.com/175580/61389924-5bf45c00-a8ec-11e9-85a7-14c647f5a2c8.png)

Will give this a pass for now and move on to other tasks. :)

Reply:
Sounds good, but we should clarify it for the sake of future replicability of the tests! :+1: